cfgov/jinja2/v1/owning-a-home/mortgage-closing/index.html

```diff
@@ -0,0 +1,162 @@
+        </a>
+      </p>
+      <h4 class="u-mb5">2. Get ready for closing</h4>
+      <p class="u-mb15">Our closing checklist lets you know what to expect at closing and what questions to ask so you will be prepared.</p>
+      <p class="u-mb30">
+        <a href="/static/pdfs/owning-a-home/checklist_mortgage_closing.pdf"
+           class="a-link
+                  a-link__jump
+                  a-link__icon
+                  a-link__no-wrap
+                  cf-icon
+                  cf-icon__after
+                  cf-icon-download">
+          <span class="a-link_text">Get our closing checklist</span>
+        </a>
+      </p>
+      <h4 class="u-mb5">3. Review the rest of your closing documents </h4>
+      <p class="u-mb15">In addition to your Closing Disclosure, there are other important documents you will need to sign at closing, such as the promissory note and the deed of trust. Get a head start on reviewing these documents by asking your closing agent for a copy of your closing packet in advance. We can help you navigate these documents.</p>
+      <p class="u-mb30">
+        <a href="/static/pdfs/owning-a-home/mortgage_closing_forms.pdf"
+           class="a-link
+                  a-link__jump
+                  a-link__icon
+                  a-link__no-wrap
+                  cf-icon
+                  cf-icon__after
+                  cf-icon-download">
+          <span class="a-link_text">Navigate your closing documents</span>
+        </a>
+      </p>
+      <h4 class="u-mb5">4. Get any errors fixed now, before closing</h4>
+      <p class="u-mb15">If you find an error in one of your closing documents, call your loan officer or settlement agent and get the error fixed right away. Even minor misspellings can delay your closing or cause big problems in the future.</p>
+      <p class="u-mb30">
+        <a href="/askcfpb/1917/what-should-i-do-if-i-find-an-error-in-one-of-my-mortgage-closing-documents.html"
+           class="a-link
+                  a-link__jump">
+          <span class="a-link_text">Learn more about what to do if you find an error</span>
+        </a>
+      </p>
+      {% include "_layouts/ratings-form.html" %}
+    </section>
+    <!-- .content_main -->
+    <aside class="content_sidebar">
+      <div class="block
+                  block__flush-top block__padded-top">
+        {{ social_media.render( {
+              "twitter_text": "About to close on a mortgage? The @CFPB’s tools and checklist help you close with confidence.",
+              "email_title": "About to close on a mortgage?",
+              "email_text": "The @CFPB’s tools and checklist help you close with confidence.",
+              "email_signature": "-- From the CFPB",
+              "linkedin_title": "About to close on a mortgage?",
+              "linkedin_text": "The @CFPB’s tools and checklist help you close with confidence. Via @CFPB"
+        } ) }}
+      </div>
+      <section class="block u-mt0">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3165/commits/adea54f8ecb4f983b08882aaf842c2e812e402bf